apps/storefront/.env.template

```diff
@@ -8,3 +8,6 @@ SCOS_BASE_URL=
 # The Cloudinary ID is not on a paid plan, which means that it cannot be heavily used. If you like to copy
 # this implementation for you production usage, make sure that you provide a reliable Cloudinary account.
 ORYX_CLOUDINARY_ID=
+
+# If set - turns on oryx-labs features
```

Comment:
can you:
- add a warning about this not being a standard supported feature?  We might already have a section in docs that you can copy

Reply:
Done (I've copied it from `libs/template/labs/src/feature.ts` file)